src/src/services/hasura/hasura.service.ts

```diff
@@ -63,6 +63,30 @@ export class HasuraService {
 		}
 	}
 
+	public async queryWithVariable(query: any) {
+		try {
+			let axios = require('axios');
+			let url = this.configService.get<string>('HASURA_BASE_URL');
+			let admin_secret = this.configService.get<string>(
+				'HASURA_ADMIN_SECRET',
+			);
+			var config_field_value = {
```

Comment:
why this name as `config_field_value` ?

Reply:
i changed name from config_field_value to config